File: cpp/apcs/1/f605/f605.cpp
#include<iostream>
#include <algorithm>
using namespace std;

int main(){
    int n,d,c=0,r=0;
    cin>>n>>d;
    int l[3];
    for(int i=0;i<n;i++){
        cin>>l[0]>>l[1]>>l[2];
        if (max({l[0],l[1],l[2]}) - min({l[0],l[1],l[2]}) >= d){
            r+=1;
            c+=(l[0]+l[1]+l[2])/3;
        }
    }
    cout<<r<<" "<<c;
    return 0;
}

File: python/apcs/1/f605.py
x,cost=0,0
n,d=map(int,input().split())
for i in range(n):
    a,b,c=map(int,input().split())
    if max(a,b,c)-min(a,b,c)>=d:
        x+=1
        cost+=sum([a,b,c])//3
print(x,cost)